loader/0000-pr-policy.md

```diff
@@ -0,0 +1,67 @@
+
+## Final Comment Period (FCP)
+- Begins immediately after the above critera is met
+- Lasts 2 days for small implementation changes, 5 days for major refactors, and 10 days for new API introductions or RFCs. Bugfixes have no FCP and are considered mergable immediately.
+- May be ended at any time by a team member if they have concerns.
+
+
+## Conflicts
+If the Loader team cannot agree on whether to merge a PR or deny it, there are steps they can take to avoid decsision paralysis. This should be a rare process--teams are encouraged to resolve these disagreements through talking the details out, but sometimes that isn't an option.
+
+In the event the disagreement is over API structure, the PR may, at the Loader team's option, be closed and an RFC be created that defines the feature.
+
+In the event the disagreement is over implementation details, or would otherwise not benefit from being forced through the long RFC process, the voting method defined below can be used.
+
+### Voting
+A vote is complete when one of the following is true:
+- Ten days has passed and the number of votes for one disposition is greater than the number of votes for the other disposition.
+- An absolute majority has been formed for one disposition.
+    - This means that even if all of the team members who have not yet voted suddenly voted for the currently losing side, the end result would not change. For example, if 7 out of a team of 10 have voted yes, the remaining 3 team members could not change the outcome of the vote.
+
+In the event of a tie, the vote is opened to the Admin Board and their votes count as 3. If the vote is still tied, the vote fails (the policy is not implemented, the PR is not closed, etc.)
+
+# Drawbacks
+- Implementing a defined policy adds a layer of beurcacy to getting PRs merged, which may be confusing.
+- There may be situations that this policy does not define, which could cause conflict or confusion within the Loader team.
+- Not having one person who has to review every PR could also lead to death by staleness, where a PR is never merged because nobody gets around to reviewing it.
+
+# Rationale and Alternatives
+We could do nothing and keep the current guessing game on when enough discusison has happened for a merge, but this could lead to PRs being rejected too quickly or merged too quickly. The latter is especially problematic because Loader is such a stable project.
+
+We could assign a few benevolant dictators to be in charge of merging PRs. At the time of writing, the most obvious choice for this would be one of the Admin Board members on the team, such as i509VCB or Haven King. However, doing so would require them to spend much more time on Loader than their other projects to avoid death by staleness, and there would be no way for the team to merge PRs if the Admins didn't feel like it.
```

Comment:
```suggestion
We could assign a few benevolent dictators to be in charge of merging PRs. At the time of writing, the most obvious choice for this would be one of the Admin Board members on the team, such as i509VCB or Haven King. However, doing so would require them to spend much more time on Loader than their other projects to avoid death by staleness, and there would be no way for the team to merge PRs if the Admins didn't feel like it.
```